enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -138,14 +139,18 @@ export const MetabaseProviderInternal = ({
     store.dispatch(setMetabaseClientUrl(authConfig.metabaseInstanceUrl));
   }, [store, authConfig.metabaseInstanceUrl]);
 
+  const instanceLocale = getSetting(store.getState(), "site-locale");
+
   return (
     <SdkContextProvider>
       <EmotionCacheProvider>
         <Global styles={SCOPED_CSS_RESET} />
         <SdkThemeProvider theme={theme}>
           <SdkFontsGlobalStyles baseUrl={authConfig.metabaseInstanceUrl} />
           <Box className={className} id={EMBEDDING_SDK_ROOT_ELEMENT_ID}>
-            <LocaleProvider locale={locale}>{children}</LocaleProvider>
+            <LocaleProvider locale={locale || instanceLocale || "en"}>
```

Comment:
Do we have tests for the fallback `"en"` when both `locale` and `instanceLocale` is undefined? I wonder if it's even possible for instance locale to be undefined.